### DISCUSSION TITLE
Source/Fuente para el repo

### DISCUSSION THREAD

**Discussion by rafacamino10** [Paco8/plugin.video.movistarplus#38] [General]
Buenas! Sería posible tener un enlace en formato fuente/source para añadir a Kodi el repo? Es que ejecuto Kodi en un Apple TV y no he logrado ni por ftp, smb, etc. enviar el .zip con el repositorio para poder instalar los archivos necesarios.
Muchas gracias de antemano!
Un saludo y tremendo trabajo

**Comment by Paco8** [comment]
¿No se puede descargar con un navegador desde el Apple TV?

**Comment by rafacamino10** [comment]
No hay navegador web en tvOS, ni un explorador de archivos para gestionar. Intenté añadir un chrome launcher a kodi desde una fuente pero tampoco, ninguno está operativo para las versiones recientes.. es un gran inconveniente que no tenga puerto usb ni posibilidad para descargar en él directamente el .zip, por eso te comentaba eso en el anterior mensaje, o alguna idea que pueda servirme de ayuda y solucionarlo. Gracias!

**Comment by Paco8** [comment]
Veré a ver qué puedo hacer.

**Comment by rafacamino10** [comment]
Perfecto, estaré atento a ver si das con alguna solución. Muchas gracias!

**Comment by Paco8** [comment]
Prueba a poner esto como fuente:
https://paco8.github.io/kodi-repo/

**Comment by rafacamino10** [comment]
Con la fuente ha ido perfecto!! He podido instalar repositorio y script, pero no contaba con que Widevine cdm no está disponible para tvOS... por lo que no me permite lanzar la emisión de video.
Te agradezco mucho la ayuda y por la pronta respuesta con la fuente.
Continuaré investigando a ver si existe alguna posibilidad para poder hacer que funcione.
Un saludo!
